examples/io/plot_read_proj.py

```diff
@@ -50,8 +50,10 @@
 mne.viz.plot_projs_topomap(empty_room_proj)
 
 ###############################################################################
-# As shown in the tutorial on how to :ref:`tut-viz-raw`
-# the ECG projections can be loaded from a file and added to the raw object
+# .. TODO: add this when the tutorial is up: "As shown in the tutorial
```

Comment:
is this still WIP?

Reply:
No, this is a reminder to add the cross ref in a later PR when the new tutorial is added.